modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -695,6 +761,102 @@ export default class AvgRTPStatsReporter {
     }
 
     /**
+     * Calculates average number of pixels for the report
+     *
+     * @param {go figure} resolution
+     * @param {boolean} isLocal if the average is to be calculated for the local
+     * video or <tt>false</tt> if for remote videos.
+     * @param {VideoType} videoType
+     * @return {number|NaN} average number of pixels or <tt>NaN</tt> if there
+     * are no samples.
+     * @private
+     */
+    _calculateAvgVideoPixels(resolution, isLocal, videoType) {
```

Comment:
it looks like `resolution` is a map?  named this way i assumed it was a single resolution.  maybe the arg should be `peerResolutions` or something?